management/post_firebase.js

```diff
@@ -0,0 +1,124 @@
+const requestPromise = require('request-promise');
+
+// [START get_sa_access_token]
+const {google} = require('googleapis');
+const SCOPES = ['https://www.googleapis.com/auth/firebase'];
+
+function getAccessToken() {
+  return new Promise(function(resolve, reject) {
+    const key = require('./service-account.json');
```

Comment:
I think this should be done at a higher scope.

Reply:
Used admin SDK to get access token so this is no longer necessary.